src/lib.rs

```diff
@@ -356,18 +335,18 @@ pub fn build_3dsx(config: &CTRConfig, verbose: bool) {
     command
         .arg(&config.target_path)
         .arg(config.path_3dsx())
-        .arg(format!("--smdh={}", config.path_smdh().to_string_lossy()));
-
-    // If romfs directory exists, automatically include it
-    let (romfs_path, is_default_romfs) = get_romfs_path(config);
-    if romfs_path.is_dir() {
-        eprintln!("Adding RomFS from {}", romfs_path.display());
-        command.arg(format!("--romfs={}", romfs_path.to_string_lossy()));
-    } else if !is_default_romfs {
-        eprintln!(
-            "Could not find configured RomFS dir: {}",
-            romfs_path.display()
-        );
+        .arg(format!("--smdh={}", config.path_smdh()));
+
+    let romfs = config
+        .romfs_dir
+        .as_deref()
+        .unwrap_or(Utf8Path::new("romfs"));
```

Comment:
I think this might actually be a change in behavior, still need to still resolve this relative to the manifest directory.

- [ ] TODO - don't merge

Reply:
O RomFS, RomFS, wherefore art thou RomFS?

--- Shakespeare, probably.